src/modules/blockchainApplication/manage/store/action.js

```diff
@@ -10,6 +10,26 @@ export const toggleApplicationPin = (chainId) => ({
   chainId,
 });
 
+/**
+ * Trigger this action to add blockchain application
+ *
+ * @returns {Object} - Action object
+ */
+export const addApplication = (application) => ({
+  type: actionTypes.addApplicationByChainId,
+  data: application,
+});
+
+/**
+ * Trigger this action to delete blockchain application
+ *
+ * @returns {Object} - Action object
+ */
+export const deleteApplication = (chainId) => ({
+  type: actionTypes.deleteApplicationByChainId,
+  data: chainId,
+});
```

Comment:
a bit wired to me passing data here shall we pass chainId
```suggestion
export const deleteApplication = (chainId) => ({
  type: actionTypes.deleteApplicationByChainId,
  chainId,
});
```